templates/about.html.tera

```diff
@@ -33,6 +33,13 @@ RewriteRule . - [R=403,L]
 if ($http_user_agent ~ PrivateBinDirectoryBot ) {
 	return 403;
 }
+			</pre>
+			<h5>Caddy</h5>
+			<pre>
+@privatebinbot {
+    header User-Agent PrivateBinDirectoryBot
```

Comment:
Given your findings in #19, does this maybe need to use a regex?
```
     header_regexp User-Agent ^PrivateBinDirectoryBot
```

Reply:
I've updated the PR to use a "fast prefix match" according to caddy docs (https://caddyserver.com/docs/caddyfile/matchers#header)

Reply:
And it works, it seems. :+1: